Modules/Core/Common/test/itkNumericTraitsTest.cxx

```diff
@@ -364,12 +350,17 @@ CheckIsComplexTraits()
                 "std::complex<float> does not have the correct IsComplex trait");
   static_assert(itk::NumericTraits<std::complex<double>>::IsComplex,
                 "std::complex<double> does not have the correct IsComplex trait");
+  static_assert(itk::NumericTraits<std::complex<long double>>::IsComplex,
+                "std::complex<long double> does not have the correct IsComplex trait");
```

Comment:
Should we really care about `long double` support? On MSVC, `long double` is basically just like `double`: 64-bits floating point.

Reply:
It is rarely used nowadays indeed. But it is still an existing type, so we can keep supporting it.

Reply:
OK, thanks.